input/reproSchemaScript/reprotofhirjson.py

```diff
@@ -0,0 +1,282 @@
+'''
+script to convert repro scheme to json for fhir questionnaire
+Not perfect conversion
+#example: "python reprotofhirjson.py <reproschema_folder> valueset"
+must be in the file structure specified in:
+https://github.com/ReproNim/reproschema-library/tree/43e7afab312596708c0ad4dfd45b69c8904088ae/activities
+
+'''
+import json
+import sys
+import os
+import logging
+from dotenv import load_dotenv
+
+load_dotenv()
+CODESYSTEM_URI = os.getenv('CODESYSTEM_URI')
+VALUESET_URI = os.getenv('VALUESET_URI')
+QUESTIONNAIRE_URI = os.getenv('QUESTIONNAIRE_URI')
+
+codeSystem_dict = dict()
+
+fhir_questionnaire = dict()
+
+fhir_valuesets = []
+fhir_codesystems = []
+
+
+
+def generate_codeSystem(options_json, linkId,questionnaire):
+   # default headers for codesystem
+    codeSystem = dict()
+    id = questionnaire + linkId
+    id = id.replace("_","-")
+    id = id.lower()
+
+    codeSystem["resourceType"] = "CodeSystem"
+    codeSystem["id"] = id
+    codeSystem["text"] = {
+    "status" : "generated",
+    "div" : "<div xmlns=\"http://www.w3.org/1999/xhtml\">Placeholder</div>"}
+
+    codeSystem["url"] = str(CODESYSTEM_URI) + id
+    codeSystem["version"] = "1.4.0"
+    codeSystem["name"] = (id).capitalize().replace("_","")
+    codeSystem["title"] = id
+    codeSystem["status"] = "active"
+    codeSystem["date"] = "2023-11-20T11:33:15-05:00"
+    codeSystem["publisher"] = "KinD Lab"
+    codeSystem["contact"] = [
+        {
+        "name" : "KinD Lab",
+        "telecom" : [
+            {
+            "system" : "url",
+            "value" : "http://fhir.kindlab.sickkids.ca"
+            }
+        ]
+        }
+    ]
+
+    codeSystem["description"] = id
+    codeSystem["caseSensitive"] = True
+    codeSystem["content"] = "complete"
+    codeSystem["count"] = len(options_json["choices"])
+    codeSystem["concept"] = []
+    
+    options = []
+    # loops through the options and appends them to the codesystem
+    for j in options_json["choices"]:
+        codeSystem_option = dict()
+        if "schema:name" in j and j["schema:name"] != "":
+            choice = j["schema:name"]
+        else:
+            choice = j["schema:value"]
+        
+        if choice and not isinstance(choice, int) and "en" in choice and isinstance(choice, dict):
```

Comment:
does this mean we are ignoring internationalization currently? should note this down somewhere